tests/Unit/Scraper/Listeners/ScrapeTest.php

```diff
@@ -51,15 +54,15 @@ public function whenConfigurationDoesNotExistItShouldThrowAnEvent(): void
             ->with($this->type)
             ->andReturn(collect());
 
-        $this->expectsEvents(InvalidConfiguration::class);
-
+            
         $scrape = new Scrape(
             $this->config,
             $this->xpathFinder,
             Log::getFacadeRoot()
         );
-
+        
```

Comment:
Same about check style. Avoid not needed spaces.

Reply:
Done